### ISSUE TITLE
OCAB Error When Installing Base

### ISSUE THREAD

**Issue by clseibold** [Skulltrail192/One-Core-API-Binaries#297]
Whenever I try to install the Base on my Windows XP machine, I keep getting this "OCAB" error, and then it seems like a lot of things revert and it finishes with an error message that "Windows has been partially updated and may not work."

**Comment by Skulltrail192** [comment]
My dear friend, without the necessary information I cannot try to help you. Please tell me:
- What version of Windows?
- Is it updated or not?
- Real or virtualized hardware?
- Do you have other programs installed?
- Are you trying to install OCA from scratch or over a previous OCA installation?
- What is the ISO used?
- Any other relevant information, including a detailed step-by-step guide on how to reproduce the problem.

**Comment by clseibold** [comment]
Windows XP Pro Service Pack 3, updated, real hardware (hp compaq dc5700 microtower).
Yes, I have a bunch of programs installed, including all of the redistributables, Visual Studio 2008, and a bunch of other stuff.

I obviously used the latest release, installing from scratch.

I click on the installer, it goes to install, and then it gives an error. Not sure what else you expect me to give you, but I cannot know unless you tell me what information you need.

**Comment by jonm58** [comment]
> Windows XP Pro Service Pack 3, updated

With POSReady Update?

**Comment by clseibold** [comment]
Yes, I think I did have the POSReady Update already installed prior to trying to install One-Core.
